Classes/Fusion/Helper/NodeInfoHelper.php

```diff
@@ -78,12 +63,6 @@ class NodeInfoHelper implements ProtectedContextAwareInterface
      */
     protected $baseNodeType;
 
-    /**
-     * @Flow\InjectConfiguration(path="userInterface.navigateComponent.nodeTree.loadingDepth", package="Neos.Neos")
-     * @var string
-     */
-    protected $loadingDepth;
-
```

Comment:
where is this used instead?

Reply:
The property is also unused in this class in Neos 7.3

However the feature still works